kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/LayoutRunner.kt

```diff
@@ -92,6 +120,31 @@ interface LayoutRunner<RenderingT : Any> {
       noinline constructor: (View) -> LayoutRunner<RenderingT>
     ): ViewBinding<RenderingT> = Binding(RenderingT::class, layoutId, constructor)
 
+    inline fun <BindingT : androidx.viewbinding.ViewBinding, reified RenderingT : Any> bind(
+      noinline bindingInflater: ViewBindingInflater<BindingT>,
```

Comment:
Passing method reference here will allow us to avoid reflection, and will make consumer side more configurable. On the other hand, it'll increase boilerplate a bit.

Reply:
Yea, not a huge fan of this reflection approach, but I'm not as worried as I'd usually be because it's reflecting on generated code that's guaranteed to have this method. But I would probably rather not merge it and only have the explicit version at least for now. It's really not that much boilerplate, and much less magic.

Reply:
So the `buildView` API passes you a `ViewGroup` parent for the view-to-inflate, but it's nullable. Unfortunately, unlike the `LayoutInflater.inflate` method, the parent parameter to `ViewBinding.inflate` is _not_ nullable. That means this binding needs to check if it's null, and decide which `inflate` method to call based on whether a container is provided or not. It's really awkward to pass two separate `inflate` references to the binding, especially since they have the same name. 

For this reason, I'm wondering if the reflection-based binding is the only reasonable one. We can do the reflection lookup lazily and cache the result, to minimize the impact on performance.